### DISCUSSION TITLE
connect esp8266 to socket.io

### DISCUSSION THREAD

**Discussion by majidkazemi2000** [socketio/socket.io#3868] [Q&A] [answered]
hi guys , How do I connect esp8266 to socket.io in node js server ? please send client code if there is a way.

**Comment by darrachequesne** [comment, marked as answer]
I'm not really familiar with ESP8266, but you should be able to use [espruino](http://www.espruino.com/) and install the JS client from NPM: http://www.espruino.com/Modules#from-npm

**Reply by majidkazemi2000** [reply]
Is there no way for Arduino?

**Reply by darrachequesne** [reply]
AFAIK, no.
